src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    border-radius: 0.5px;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+}
+
+.stars--1 :nth-child(-n+1),
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4),
+.stars--5 :nth-child(-n+5) {
```

Comment:
try to use [cycle for](https://sass-lang.com/documentation/at-rules/control/for)